[2021-10-02 19:28:14,548] {taskinstance.py:903} INFO - Dependencies all met for <TaskInstance: my_dag.Training_Model_A 2021-10-02T19:28:13.149551+00:00 [queued]>
[2021-10-02 19:28:14,585] {taskinstance.py:903} INFO - Dependencies all met for <TaskInstance: my_dag.Training_Model_A 2021-10-02T19:28:13.149551+00:00 [queued]>
[2021-10-02 19:28:14,587] {taskinstance.py:1095} INFO - 
--------------------------------------------------------------------------------
[2021-10-02 19:28:14,588] {taskinstance.py:1096} INFO - Starting attempt 1 of 1
[2021-10-02 19:28:14,590] {taskinstance.py:1097} INFO - 
--------------------------------------------------------------------------------
[2021-10-02 19:28:14,621] {taskinstance.py:1115} INFO - Executing <Task(PythonOperator): Training_Model_A> on 2021-10-02T19:28:13.149551+00:00
[2021-10-02 19:28:14,635] {standard_task_runner.py:52} INFO - Started process 20803 to run task
[2021-10-02 19:28:14,643] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'my_dag', 'Training_Model_A', '2021-10-02T19:28:13.149551+00:00', '--job-id', '37', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/my_dag.py', '--cfg-path', '/tmp/tmpatnbrw2p', '--error-file', '/tmp/tmps9t4irbv']
[2021-10-02 19:28:14,645] {standard_task_runner.py:77} INFO - Job 37: Subtask Training_Model_A
[2021-10-02 19:28:14,709] {logging_mixin.py:109} INFO - Running <TaskInstance: my_dag.Training_Model_A 2021-10-02T19:28:13.149551+00:00 [running]> on host d6b4131456ad
[2021-10-02 19:28:14,788] {taskinstance.py:1254} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=my_dag
AIRFLOW_CTX_TASK_ID=Training_Model_A
AIRFLOW_CTX_EXECUTION_DATE=2021-10-02T19:28:13.149551+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2021-10-02T19:28:13.149551+00:00
[2021-10-02 19:28:14,814] {python.py:151} INFO - Done. Returned value was: None
[2021-10-02 19:28:14,832] {taskinstance.py:1219} INFO - Marking task as SUCCESS. dag_id=my_dag, task_id=Training_Model_A, execution_date=20211002T192813, start_date=20211002T192814, end_date=20211002T192814
[2021-10-02 19:28:14,865] {local_task_job.py:151} INFO - Task exited with return code 0
[2021-10-02 19:28:14,937] {local_task_job.py:261} INFO - 0 downstream tasks scheduled from follow-on schedule check
